test/bigtest/interactors/package-content.js

```diff
@@ -0,0 +1,37 @@
+import {
+  interactor,
+  clickable,
+  fillable,
+  isPresent,
+  selectable,
+  text,
+} from '@bigtest/interactor';
+
+export default @interactor class PackageContentInteractor {
```

Comment:
This interactor is waaaaaay overloaded with responsibilities. This Package Content interactor has methods related to, yes, the package contents, but also managing the Basket, filling out Agreement forms, and interacting with an Agreement's view?

Reply:
There's still too much stuff in here. We're still having a `PackageContentInteractor` responsible for interacting with things that aren't even in its _route_ (`/erm/eresources` vs `/erm/eresources/{pkgId}`).

Honestly, it looks like the only things here that should exist in this interactor are the `click***` actions. They're the only things here that are truly part of the `PackageContents` component.

Reply:
While you're at it, you may as well add the `clickAll` and `clickCurrent` handlers though 🤷‍♂